lib/tasks/openapi_generate.rake

```diff
@@ -0,0 +1,388 @@
+class OpenapiGenerator
```

Comment:
I see most of this is just a copy of the existing swagger generator.  I'm concerned about duplication, but that can be cleaned up in a follow up.  Even so, can you show a diff between the two so I can figure out how to review this?

Reply:
I think this is now better, I've essentially git mv'd the original swagger_generate.rake to the new openapi_generate.rake successor, so git diffs now show the changes.